src/typings/win-ca.d.ts

```diff
@@ -86,7 +86,7 @@ declare module 'win-ca' {
     }
   }
 
-  function api(params?: apiOptions & { genarator?: false, format?: _der2.format }): void;
+  function api(params?: apiOptions & { generator?: false, format?: _der2.format }): void;
```

Comment:
??

Reply:
This is a correct fix.  Typescript overload declaration that's incorrect (but was okay because we didn't end up using this case).